### ISSUE TITLE
Creating a new layout gives an error.

### ISSUE THREAD

**Issue by thaoms** [engram-design/CPNav#38]
Latest Craft Release 3.0.1
Latest CP Nav

Console output:
```
{error: "Calling unknown method: craft\elements\db\UserQuery::client()"}
error: "Calling unknown method: craft\elements\db\UserQuery::client()"
```

**Comment by chillibe** [comment]
The solution for this issue is situated here : LayoutController.php
Line 52

```
if (Craft::$app->getEdition() == Craft::Client) {
     $variables['clientAccount'] = User::find()->client()->status(null)->one();
 } else if (Craft::$app->getEdition() == Craft::Pro) {
      $variables['allGroups'] = Craft::$app->userGroups->getAllGroups();
}
```

Thank you

**Comment by johnhoysa** [comment]
I have the same issue I think. 
Craft - 3.0.1
CP Nav 2.0.3

My chunk of code looks identical to what you have, and unfortunately your suggestion did not fix my issue. 

Could it be because I am using a trial version of Craft within MAMP? PHP version 7.0.22

My Base URL is @web/

Console error:
Failed to load resource: the server responded with a status of 500 (Internal Server Error)
http://localhost:8888/myradical-cms/web/index.php?p=admin/actions/cp-nav/layout/get-hud-html
{
    "error": "Calling unknown method: craft\\elements\\db\\UserQuery::client()"
}

Thanks.

**Comment by engram-design** [comment]
I believe this is an issue caused by the 3.0.0 release removing support for the Craft Client edition. Will issue a fix soon.

**Comment by engram-design** [comment]
This should be fixed in [2.0.4](https://github.com/verbb/cp-nav/releases/tag/2.0.4).
